### PR TITLE
Edge section and link restructure

### PR DESCRIPTION
**Pull request by dianaolives**

### CODE REVIEW

File: content/edge/edge-introduction-bundle/iot-platform.md [hunk 1]
```diff
@@ -8,57 +8,57 @@ section:
...
+|[Managing users](/standard-tenant/managing-users/)|How to [create users](/standard-tenant/managing-users/#to-add-a-user), edit, disable or delete them.
+|[Managing permissions](/standard-tenant/managing-permissions/)|How to create and edit [global roles](/standard-tenant/managing-permissions/#global-roles) and [inventory roles](/standard-tenant/managing-permissions/#inventory-roles), how to assign them to users, and how to [grant application access](/standard-tenant/managing-permissions/#application-access).
+|[Managing the ecosystem](/standard-tenant/ecosystem/)|How to manage and [configure own applications](/standard-tenant/ecosystem/#managing-applications) in your {{< product-c8y-iot >}}  Edge account.
+|[Changing settings](/standard-tenant/changing-settings/)|How to change account settings like [application settings](/standard-tenant/changing-settings/#application) or **does this still exist? ->**[password policy settings](/users-guide/administration#changing-password-settings) and how to manage the [properties library](/standard-tenant/changing-settings/#properties-library).
```
Comment (dianaolives) [inline content/edge/edge-introduction-bundle/iot-platform.md:63]: I commented in the text if the "password policy settings" section still exists?

File: content/edge/edge-introduction-bundle/iot-platform.md [hunk 2]
```diff
@@ -8,57 +8,57 @@ section:
 
 Since {{< product-c8y-iot >}} Edge is based on the same software as the cloud-based {{< product-c8y-iot >}} platform version, the included applications are the same in both versions, with minor restrictions.
 
-For general information about functionalities and features of {{< product-c8y-iot >}} platform, see [{{< product-c8y-iot >}} platform](/welcome/intro-documentation/).
+For general information about functionalities and features of {{< product-c8y-iot >}} platform, see [Getting started](/get-familiar-with-the-ui/ui-introduction/).
```
Comment (BeateRixen) [inline content/edge/edge-introduction-bundle/iot-platform.md:11]:
```suggestion
For general information about functionalities and features of {{< product-c8y-iot >}} platform, see [Get familiar with the UI](/get-familiar-with-the-ui/ui-introduction/).
```

File: content/edge/edge-introduction-bundle/iot-platform.md [hunk 3]
```diff
@@ -8,57 +8,57 @@ section:
...
+|[Configuring {{< product-c8y-iot >}} Edge](/edge/edge-configuration/)|Providing information on Edge specific configuration options.
+|[Home screen](/standard-tenant/home-screen/)|Providing information on your capacity usage and subscribed applications.
+|[Managing users](/standard-tenant/managing-users/)|How to [create users](/standard-tenant/managing-users/#to-add-a-user), edit, disable or delete them.
+|[Managing permissions](/standard-tenant/managing-permissions/)|How to create and edit [global roles](/standard-tenant/managing-permissions/#global-roles) and [inventory roles](/standard-tenant/managing-permissions/#inventory-roles), how to assign them to users, and how to [grant application access](/standard-tenant/managing-permissions/#application-access).
+|[Managing the ecosystem](/standard-tenant/ecosystem/)|How to manage and [configure own applications](/standard-tenant/ecosystem/#managing-applications) in your {{< product-c8y-iot >}}  Edge account.
```
Comment (BeateRixen) [inline content/edge/edge-introduction-bundle/iot-platform.md:62]:
```suggestion
|[Managing applications](/standard-tenant/ecosystem/)|How to manage and [configure own applications](/standard-tenant/ecosystem/#managing-applications) in your {{< product-c8y-iot >}}  Edge account.
```

File: content/edge/edge-introduction-bundle/iot-platform.md (hunk 1 above)
Comment (BeateRixen) [reply content/edge/edge-introduction-bundle/iot-platform.md:63]: This link does no longer exist. Just remove. The entire tables which refer to the former user guide are a bit outdated and could need an update. Let´s do that in a separate step. @sharath-jk can you please take a look? I assume that several new chapters are missing in these lists. But let´s wait until this PR is merged.

File: content/edge/edge-update-bundle/migration-107-to-109.md [hunk 4]
```diff
@@ -0,0 +1,325 @@
+---
+weight: 15
+title: Migrating Cumulocity IoT Edge 10.7 to 10.9
```
Comment (BeateRixen) [inline content/edge/edge-update-bundle/migration-107-to-109.md:3]: @sharath-jk Do we really still need this chapter in the documentation after 10.18?

File: content/edge/operating-edge.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+title: Cumulocity IoT Edge Operations
```
Comment (BeateRixen) [inline content/edge/operating-edge.md:2]:
```suggestion
title: Cumulocity IoT Edge operations
```

File: content/edge/edge-rest-api.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+---
+weight: 90
+title: REST APIs for Edge
```
Comment (BeateRixen) [inline content/edge/edge-rest-api.md:3]: @sharath-jk Could we move this to the introduction section? As it does not hold much content, I would actually not have an extra chapter for this.

File: content/edge/edge-introduction-bundle/connecting-devices.md [hunk 7]
```diff
@@ -6,15 +6,15 @@ section:
   - edge_server
```
Comment (BeateRixen) [inline content/edge/edge-introduction-bundle/connecting-devices.md:6]: @sharath-jk We have an extra section now "Connecting devices to Cumulocity Iot Edge". Shouldn´t we integrate this content there or at least add a link here?

File: content/edge/edge-introduction-bundle/iot-platform.md (hunk 1 above)
Comment (sharath-jk) [reply content/edge/edge-introduction-bundle/iot-platform.md:63]: @BeateRixen, @dianaolives - Most of the tables in the Introduction section have been copied from Core documentation. If its outdated, then I'll have to update the tables. Please let me know if we need to use the table available here:
https://cumulocity.com/guides/users-guide/administration/#overview

File: content/edge/edge-update-bundle/migration-107-to-109.md (hunk 4 above)
Comment (sharath-jk) [reply content/edge/edge-update-bundle/migration-107-to-109.md:3]: @BeateRixen - I'll have to check with Bhaskar.

File: content/edge/edge-rest-api.md (hunk 6 above)
Comment (sharath-jk) [reply content/edge/edge-rest-api.md:3]: @BeateRixen - when we restructured the Edge documentation, we wanted it as a separate chapter. But we can discuss this with Bhaskar and then decide.

File: content/edge/edge-introduction-bundle/connecting-devices.md (hunk 7 above)
Comment (sharath-jk) [reply content/edge/edge-introduction-bundle/connecting-devices.md:6]: Sure, @BeateRixen. This we can do.

File: content/edge/edge-rest-api.md (hunk 6 above)
Comment (BeateRixen) [reply content/edge/edge-rest-api.md:3]: OK, then let´s leave it for now and decide when we also move the Edge Kubernetes guide.

File: content/edge/edge-update-bundle/migration-107-to-109.md (hunk 4 above)
Comment (BeateRixen) [reply content/edge/edge-update-bundle/migration-107-to-109.md:3]: Same here, let´s leave it for now.

File: content/edge/edge-introduction-bundle/iot-platform.md (hunk 1 above)
Comment (BeateRixen) [reply content/edge/edge-introduction-bundle/iot-platform.md:63]: We do no longer maintain the tables in the core documentation. This table is outdated for the CD version, we do not even have a user guide here. Diana adapted the links but you would have to compare this with the current structure to see if it is complete.

File: content/edge/edge-introduction-bundle/connecting-devices.md (hunk 7 above)
Comment (BeateRixen) [reply content/edge/edge-introduction-bundle/connecting-devices.md:6]: I have moved this file and integrated it into the Connecting devices chapter, and modified the links to Modbus and OPC UA to no longer point to the protocol integration section but to the respective Edge chapters.

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**